custom_components/tuya_local/devices/kogan_lx10_vacuum.yaml

```diff
@@ -16,8 +16,10 @@ primary_entity:
       type: string
       mapping:
         - value: start
+          dps_val: true
```

Comment:
The type is string, it cannot take a value of true.

It seems you actually want a conditional mapping here, but there are two other dps that you want to be conditional on, which is beyond the capabilities of the mapping logic.

"start" can probably be renamed to "activate", which is already handled in code.  Then you only have "pause" that you need to be conditional on.